app/src/main/java/com/framgia/fsalon/screen/booking/BookingFragment.java

```diff
@@ -1,14 +1,17 @@
 package com.framgia.fsalon.screen.booking;
 
+import android.app.Activity;
 import android.databinding.DataBindingUtil;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
```

Comment:
ERROR: Unused import: android.support.v7.app.AppCompatActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports